src/Templates/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# Test apps
+TestMaui/
```

Comment:
The `build.ps1` script in this folder generates this test output folder, so this helps avoid checking it in by accident.